docs/src/content/docs/getting-started/concepts.mdx

```diff
@@ -29,34 +34,72 @@ Repository
   |-- Directory 2
         |-- Workspace: default
 ```
-
+:::note
+Terrateam executes operations like plan and apply within the context of a Dirspace.
```

Comment:
"Terrateam executes" appears three times in the text. You could mention it once, then reference it in the later sections.